Assets/Scripts/Camera/SwapCameraOnEnter.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+namespace Camera
+{
+    [RequireComponent(typeof(BoxCollider))]
+    public class SwapCameraOnEnter : MonoBehaviour
+    {
+        [SerializeField] private GameObject cameraToSwapTo;
+        [SerializeField] private GameObject[] camerasToDisable;
+
+        private void OnTriggerEnter(Collider other)
+        {
+            if (!other.gameObject.CompareTag("Player")) return;
+            
+            Debug.Log("player enter");
```

Comment:
I would have the debug log what object it is entering, as it currently is, it doesn't offer much useful information.

Reply:
that debug is to be removed, was used for quick testing